tests/utils.py

```diff
@@ -0,0 +1,36 @@
+import requests
+from decouple import config
+
+TEST_URL = config("TEST_URL", default="https://referite-6538ffaf77b0.herokuapp.com/")
+
+def get_handler(url, token):
+    """GET url with authorization token"""
+    return requests.get(
+        f'{TEST_URL}{url}',
+        headers={
+            'authorization': token
+        }
+    )
+
+def login_post_handler(url, data):
+    """POST form data to url without authorization token"""
+    return requests.post(
+        f'{TEST_URL}{url}',
+        data=data,
+        headers={
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+    )
+
+def data_post_handler(url, token, data=None):
```

Comment:
why  don't initial data to empty dict instead of giving it `None` and write condition to check later?

Reply:
its the same

Reply:
same but its make code cleaner

Reply:
why

Reply:
its remove 2 line of code

Reply:
![image](https://github.com/Referite/Referite-Backend/assets/93331101/36eab7a6-e2ef-48fa-9927-1780543d63e9)

Reply:
it can remove**

Reply:
what if data is None?

Reply:
umm.... ช่างแม่ง for sourcery, Just default to dictionary

Reply:
we shouldn't post None data isn't it? If I remove 2line of code, It can post none data now

Reply:
yep its make sense